pytest_reportportal/plugin.py

```diff
@@ -103,7 +54,7 @@ def pytest_unconfigure(config):
 def pytest_addoption(parser):
     group = parser.getgroup("reporting")
     group.addoption(
-        "--rp-launch",
+        '--rp-launch',
```

Comment:
What is the reason to change quotes here and keep double quotes in other places?

Reply:
Fixed